src/OrchardCore/OrchardCore.Infrastructure.Abstractions/OrchardCore.Infrastructure.Abstractions.csproj

```diff
@@ -17,6 +17,7 @@
   <ItemGroup>
     <ProjectReference Include="..\OrchardCore.Abstractions\OrchardCore.Abstractions.csproj" />
     <ProjectReference Include="..\OrchardCore.Data.Abstractions\OrchardCore.Data.Abstractions.csproj" />
+    <ProjectReference Include="..\OrchardCore.ResourceManagement.Core\OrchardCore.ResourceManagement.Core.csproj" />
```

Comment:
This is the first time I saw Core referenced in abstractions project, seems odd :(

Reply:
I fixed this by moving the Options model to the abstraction class